ledger-service/http-json/src/main/scala/com/digitalasset/http/query/ValuePredicate.scala

```diff
@@ -369,11 +316,8 @@ object ValuePredicate {
             val elemTy = soleTypeArg("Optional")
             fromOptional(q, elemTy)
         }
-        case TextMap => {
-          case JsObject(q) =>
-            val elemTy = soleTypeArg("Map")
-            MapMatch(SortedLookupList(q) mapValue (fromValue(_, elemTy)))
-        }
+        case List | TextMap =>
+          predicateParseError(s"${typ.typ} not supported")
         case GenMap =>
           // FIXME https://github.com/digital-asset/daml/issues/2256
```

Comment:
the `FIXME` is not valid anymore. Does not make much sense to support `GenMap` queries when we are dropping `TextMap` query support.

Reply:
Combined in 4061ffc